cmd/regctl/image.go

```diff
@@ -1383,51 +1386,78 @@ func (imageOpts *imageCmd) runImageCreate(cmd *cobra.Command, args []string) err
 }
 
 func (imageOpts *imageCmd) runImageExport(cmd *cobra.Command, args []string) error {
+	var output string
+	// Legacy behavior: if no output is specified, use the second argument as the output
+	if imageOpts.output == "" && len(args) == 2 {
+		output = args[1]
+		args = args[:1]
+	} else if imageOpts.output == "" || imageOpts.output == "-" {
+		if out, ok := cmd.OutOrStdout().(interface{ Fd() uintptr }); ok {
+			if term.IsTerminal(int(out.Fd())) {
```

Comment:
I noticed that, but there are two other copies of this same test, one of them in ascii package itself. It did not seem like there is intention to have this trivial test implemented only once let alone imported from a package that seems to be for another purpose.

Reply:
And this is not really related to fixing this issue. I just got annoyed I made a mistake testing and it wrote the tar to tty. I don't think there is any use case in writing a binary file to tty since tty would anyway break it. I can remove it if someone expects to be able to write a tar to tty.

Reply:
I notice the ascii was already imported for the same purpose.